static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issueSummary.tsx

```diff
@@ -74,7 +74,9 @@ function IssueTitle(props: IssueTitleProps) {
     <TitleWithLink
       {...commonEleProps}
       to={{
-        pathname: `/organizations/${organization.slug}/issues/${props.data.id}/events/${props.event_id}/`,
+        pathname: props.event_id
```

Comment:
Worth adding a comment as to how the fallback works here, future readers wont know exactly what these links point to